src/commands/mount.rs

```diff
@@ -0,0 +1,101 @@
+//! `mount` subcommand
+mod fusefs;
+
+use std::{ffi::OsStr, path::PathBuf};
+
+use crate::{repository::CliIndexedRepo, status_err, Application, RUSTIC_APP};
+
+use abscissa_core::{Command, Runnable, Shutdown};
+use anyhow::Result;
+use fuse_mt::{mount, FuseMT};
+use rustic_core::vfs::{IdenticalSnapshot, Latest, Vfs};
+
+use fusefs::FuseFS;
```

Comment:
```suggestion
use crate::commands::mount::fusefs::FuseFS;
```

then we can merge it with the crate import on top as well

Reply:
Actually I don't see why this should be changed - having an import of a submodule is quite standard, IMO. I just moved it up to the `use` statement.

Reply:
Just for consistency to always import from the crate root, because we do that everywhere I think